Project-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```
@@ -1 +1,45 @@
 /* All styles for your header Components live here */
+
+.header{
+    
+    width:100%;
+    
+    display:flex;
+    flex-wrap:wrap;
+    flex-direction: row;
+}
+
+.title{
+    
+   
+    width:60%;
+    height: 3rem;
+    display:flex;
+   
+}
+.lambda-1{
+    width: 100%;
+ 
+}
+.lambda-2{
+    width:100%;
+  
+}
+.lambda-3{
+width:100%;
+padding:1.5rem;
+font-size:1.5rem;
+
+}
+.content{
+
```

```suggestion
margin-left: 10%;
```
Now that we know that the .thumb-img class has a width of 10%, we know that's how far to the right .content should go. You should add this margin to .card-content and .card-img as well